test/plugin/index.spec.js

```diff
@@ -1,5 +1,8 @@
 "use strict";
 
+const sinon = require("sinon");
```

Comment:
GLOBAL: Don't use `sinon` as a best practice. Instead get spys and stubs from `base.sandbox`.